libpod/healthcheck.go

```diff
@@ -154,7 +142,7 @@ func (c *Container) runHealthCheck(ctx context.Context, isStartup bool) (define.
 		}
 	}
 
-	eventLog := strings.Join(stdout, "\n")
+	eventLog := output.String()
```

Comment:
Should we ensure this is newline-terminated?

Reply:
(As in, even if the app in the `podman exec` didn't bother adding a trailing newline, I think we should, for the sake of clean logs)

Reply:
No docker doesn't add one and I think we should keep the output as we got it. And I don't see why logs would be any cleaner with our without it. Normally you see this in podman inspect which always adds a newline on the cli level see the e2e test. An otherwise you just see the log as json so the newline will be escaped in the string there.

Reply:
OK